docs/reference/search/explain.asciidoc

```diff
@@ -1,16 +1,84 @@
 [[search-explain]]
 === Explain API
 
-The explain api computes a score explanation for a query and a specific
+Returns information about why a specific document matches (or doesn't match) a 
+query.
+
+[source,console]
+--------------------------------------------------
+GET /twitter/_explain/0
+{
+      "query" : {
+        "match" : { "message" : "elasticsearch" }
+      }
+}
+--------------------------------------------------
+// TEST[setup:twitter]
+
+
+[[sample-api-request]]
+==== {api-request-title}
+
+`GET /<index>/_explain/<id>` +
```

Comment:
```suggestion
`GET /<index>/_explain/<id>`
```

Current version makes these display on the same line:
http://elasticsearch_46857.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/search-explain.html#sample-api-request